README_DataTeam.md

```diff
@@ -0,0 +1,17 @@
+# Guide for the Data Team
+
+This is a short Readme specific to Data Team (MSFC) tasks, and usage of Zenhub.
+
+## Zenhub
+
+To make full use of Zenhub you need to install the browser extension. This will allow you to the Workspaces under the Zenhub tab in each repo.
+
+## Issues
+
+You can file tickets either from the Zenhub DataTeam workspace, or on the respective github repo the issue relates to. The workspace board will aggregate the tickets for viewing in a single board.
+
+If the issue is general to the platform, you aren't sure which repo it belongs in, or is coming from a platform **user** file it under the Zenhub repo, and add appropriate labels. If it concerns the data team label **MSFC**, use as many labels as seems relevant.
+
+## Data Requests
+
+End users for now will be encouraged to file tickets in the [Zenhub](https://github.com/MAAP-Project/ZenHub) repo. This includes requests for upgrades to, or new data. For data requests, users should select the data request template.
```

Comment:
suggested wording:

>This includes requests for upgrades to existing data or new data.

Reply:
Can we link to the data request template?

Reply:
Yes, once it exists should be
https://github.com/MAAP-Project/ZenHub/issues/new?template=data_request.md
Will add that to text, might need to check and adjust after merge.